### ISSUE TITLE
multiple regressions since 0.40.0

### ISSUE THREAD

**Issue by theofficialgman** [v1cont/yad#240]
I realize this isn't the best bug report since it includes multiple regressions but I do not want to keep track of these critical issues separately.

I took some time to triage... basically yad has been broken in many ways ever since 0.41.0 (that includes 1.0, 2.0, 12.1, etc), 0.40.0 is the LAST version to work without any major problems.
to give examples of what is broken in 0.41.0+:
- window positioning
- window sizing
- windows pop in and move from the top left to the position specified

(not all of this is on XORG or XWAYLAND where window positioning and sizing are expected to work and functioned previously)

sizing seems correct in the latest release, but as I mentioned before, positioning is still broken.
all of this is with a single monitor... multimonitor is apparently more broken in the latest versions according to the github issues https://github.com/v1cont/yad/issues/113 https://github.com/v1cont/yad/issues/15

I guess it is a good reason that a lot of distros have not "updated" since 0.40.0 since it would be a significant downgrade due to these bugs

**Comment by theofficialgman** [comment]
CC: @bgermann
```
yad (7.2-0.1) experimental; urgency=medium

  * Non-maintainer upload.
  * Update debhelper to compat 10.
  * Switch to GitHub upstream and import new version (Closes: #1029395).
  * Salvage packgage (Closes: #1024523).

 -- Bastian Germann <bage@debian.org>  Tue, 30 May 2023 16:27:22 +0200
 ```
 please do not move any version higher than 0.40.0 to testing or stable until these bugs are resolved. I do not want to have to ship custom non-broken yad on debian and ubuntu stable releases.

**Comment by nerun** [comment]
@theofficialgman,  are you the Debian maintainer of YAD package? Why 7.2 is in the experimental instead of 13.0? Is it more stable than 13.0?

**Comment by bgermann** [comment]
You can see the original request by Alexis Puente Montiel for the experimental update in [Debian bug 1024523](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1024523#40). I guess he handed in this version because it is in Pica Libre. I have sponsored it but as this issue was brought up, nobody bothered to upload it to Debian unstable.

**Comment by theofficialgman** [comment]
> @theofficialgman, are you the Debian maintainer of YAD package? Why 7.2 is in the experimental instead of 13.0? Is it more stable than 13.0?

No. Just looking out yad as a dependency that projects I contribute on and don't want a version with these issues making it's way into stable or Ubuntu releases.

I pinged the maintainer and their response is as above.

**Comment by johnraff** [comment]
> I realize this isn't the best bug report since it includes multiple regressions but I do not want to keep track of these critical issues separately.

> please do not move to any version higher than 0.40.0 to testing or stable until these bugs are resolved


Has each of these bugs been reported here on GitHub? How can users and packagers monitor progress otherwise?

**Comment by bgermann** [comment]
Ubuntu has imported Debian's experimental 7.2. No bug reports for a year that would resemble this one.
